candi/bashible/bundles/ubuntu-lts/all/010_disable_unattended_upgrades.sh.tpl

```diff
@@ -12,12 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Disable auto reboot and remove unused deps
-if [ -f "/etc/apt/apt.conf.d/50unattended-upgrades" ] ; then
-  sed -i 's/\/\/Unattended-Upgrade::Automatic-Reboot "false"/Unattended-Upgrade::Automatic-Reboot "false"/g' /etc/apt/apt.conf.d/50unattended-upgrades
-  sed -i 's/\/\/Unattended-Upgrade::InstallOnShutdown "true"/Unattended-Upgrade::InstallOnShutdown "false"/g' /etc/apt/apt.conf.d/50unattended-upgrades
-  sed -i 's/\/\/Unattended-Upgrade::Remove-Unused-Dependencies "false"/Unattended-Upgrade::Remove-Unused-Dependencies "false"/g' /etc/apt/apt.conf.d/50unattended-upgrades
-fi
-
-# Disable periodic unattended-upgrades
-sed -i 's/APT::Periodic::Unattended-Upgrade "1"/APT::Periodic::Unattended-Upgrade "0"/g' /etc/apt/apt.conf.d/*
+apt-get purge unattended-upgrades -y
```

Comment:
```root@dev-master-0:~# apt-get purge unattended-upgrades -y
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages were automatically installed and are no longer required:
  libjq1 libonig5
Use 'apt autoremove' to remove them.
The following packages will be REMOVED:
  unattended-upgrades*
0 upgraded, 0 newly installed, 1 to remove and 128 not upgraded.
1 not fully installed or removed.
After this operation, 451 kB disk space will be freed.
(Reading database ... 71512 files and directories currently installed.)
Removing unattended-upgrades (2.3ubuntu0.1) ...
Processing triggers for man-db (2.9.1-1) ...
(Reading database ... 71485 files and directories currently installed.)
Purging configuration files for unattended-upgrades (2.3ubuntu0.1) ...
dpkg: warning: while removing unattended-upgrades, directory '/var/log/unattended-upgrades' not empty so not removed
Processing triggers for systemd (245.4-4ubuntu3.13) ...
W: APT had planned for dpkg to do more than it reported back (4 vs 7).
   Affected packages: containerd.io:amd64
```
it is a cosmetic warning, but can we supress them ?

Reply:
Why would we want to remove helpful warnings? They might help if something goes wrong.